CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 # CHANGELOG
 
+## v0.17.0 (2024-08-01)
 
+### Feature
+
+* Add support for [upload_document](https://elis.rossum.ai/api/docs/#create-upload)
+  * Includes models for [Task](https://elis.rossum.ai/api/docs/#task), [Upload](https://elis.rossum.ai/api/docs/#upload)
+  * New poll_task method to work with Tasks
+  
+### Chore
+
+* Allow passing extra request params with fetch_one (to disable redirects for Tasks)
+* Allow using enums in dataclass models
+* Touch up e2e test file
```

Comment:
Please don't write the changelog by hand, it's done automatically by semantic release.

Reply:
I assumed a base changelog is generated and then it can be edited for improvements, since we have been writing commits differently than changelogs in EXE.